docs/kubernetes_basic.md

```diff
@@ -132,3 +132,29 @@ my-app-a9f44174fa720b22-exec-3   1/1     Running   0          89s
 
 The moment we close the Clojure REPL session, the worker pods get deleted in Kubernetes.
 
+## Reading files into Spark on Kubernetes
+
+We have a lot of different options for reading files into Spark in a Kubernetes setup.
+The main requirement is, that the files can be addressed under a common URL in the driver and all workers.
+
+In Geni all file reading happens via functions of form  `g/read_xxx_!`. We can use different (eventually remote) file sytems by using different forms of the URL (specifically by using different prefixes).
+
+The reading of  files by Geni can be achieved in 4 different ways:
+
+0. (Just copy the data files into the same path in all the Kubernetes pods and the driver machine. This is very adhoc and likely not suitable in most scenarios, as the data files would be lost at the end of the REPL session due to pods being terminated.)
```

Comment:
Let's leave the parentheses out. The points being made here are actually very important.